docs/jobs.md

```diff
@@ -39,11 +39,21 @@ import socket
 def add(x, y):
     job = get_current_job()
     job.meta['handled_by'] = socket.gethostname()
-    job.save()
+    job.update_meta()
+    
+    # do more work
+    time.sleep(1)
     return x + y
 {% endhighlight %}
 
-
+<div class="warning">
+    <img style="float: right; margin-right: -60px; margin-top: -38px" src="{{site.baseurl}}img/warning.png" />
+    <strong>Warning!</strong>
+    <p>
+The `.update_meta()` was introduced in RQ 0.7.2. You can use the `.save()` method instead 
```

Comment:
Instead of mentioning that update_meta was introduced in 0.8, I think we should just change the "new in version 0.3.3" to "improved in 0.8.0".